### ISSUE TITLE
Small mistake in tutorials/02

### ISSUE THREAD

**Issue by klememi** [MI-AFP/tutorials#5]
In the tutorial `02_functions-types.md`

> ```haskell
> factorial n = fac' n 1 
>             where fac' 0 acc = acc
>                   fac' x acc = fac' (x - 1) (x * acc) 
> ```
> 
> 
> 1. `factorial 3`
> 2. `fac' 3 1`
> 3. `fac' 2 3`
> 4. `fac' 1 6`
> 5. `fac' 1 6`
> 6. `6`

step 5. should be `fac' 0 6` if I got it right.

**Comment by MarekSuchanek** [comment]
Good catch! 👍 Will be fixed... (I plan to add some link to diagram of basic typeclasses as well.)
